website/cue/reference/releases/0.47.0.cue

```diff
@@ -0,0 +1,427 @@
+				"""
+			contributors: ["ynachi"]
+		},
+		{
+			type: "fix"
+			description: """
+				Fixed a `kubernetes source` bug where `use_apiserver_cache=true` but there is no `resourceVersion=0` parameters in list request. As [the issue](https://github.com/kube-rs/kube/issues/1743) mentioned, when there are `resourceVersion =0` and `!page_size.is_none` in the `ListParams` fields, the parameter `resourceVersion=0` will be ignored by `kube-rs` sdk. If no parameter `resourceVersion` passed to the apiserver, the apiserver will list pods from ETCD instead of in memory cache.
+				"""
+			contributors: ["xiaozongyang"]
+		},
+		{
+			type: "feat"
+			description: """
+				Added `timeout` config option to the `healthcheck` sink configuration. Previously it was hardcoded to 10 seconds across all components, but now it can be configured per component.
+				"""
+			contributors: ["esensar", "Quad9DNS"]
+		},
+		{
+			type: "enhancement"
+			description: """
+				The [enrichment functions](https://vector.dev/docs/reference/vrl/functions/#enrichment-functions) now support bounded date range filtering using optional `from` and `to` parameters. There are no changes to the function signatures.
+				"""
+			contributors: ["nzxwang"]
+		},
+		{
+			type: "feat"
+			description: """
+				Added `wildcard_matching` global config option to set wildcard matching mode for inputs. Relaxed mode allows configurations with wildcards that do not match any inputs to be accepted without causing an error.
+				
+				Example config:
+				
+				```yaml
+				wildcard_matching: relaxed
+				
+				sources:
+				  stdin:
+				    type: stdin
+				
+				# note - no transforms
+				
+				sinks:
+				  stdout:
+				    type: console
+				    encoding:
+				      codec: json
+				    inputs:
+				      - "runtime-added-transform-*"
+				
+				```
+				"""
+			contributors: ["simplepad"]
+		},
+		{
+			type: "fix"
+			description: """
+				Fixed a bug that allowed a record to be transmitted with an IPv6 prefix larger than allowed (128) in `dnsmsg_parser::DnsMessageParser` parse functions.
+				"""
+			contributors: ["wooffie"]
+		},
+		{
+			type: "fix"
+			description: """
+				Added checks for integer operations with timestamps which used in `dnstap-parser::DnstapParser::parse`.
+				"""
+			contributors: ["wooffie"]
+		},
+		{
+			type: "enhancement"
+			description: """
+				Files specified in the `file` and `files` fields of `remap` transforms are now watched when `--watch-config` is enabled. Changes to these files automatically trigger a configuration reload, so there's no need to restart Vector.
+				"""
+			contributors: ["nekorro"]
+		},
+		{
+			type: "enhancement"
+			description: """
+				The `amqp` sink now supports setting the `priority` for messages. The priority value can be templated to an integer between 0 and 255 (inclusive). Also, this new field is template-able and Vector templates can now render numerical values.
```

Comment:
> Vector templates can now render numerical values

Suggestion: this seems like a big change that could be mentioned as a separate enhancement.